lib/cpl.rb

```diff
@@ -181,7 +181,7 @@ def self.all_base_commands
         begin
           config = Config.new(args, options)
 
-          show_info_header(config) if with_info_header
+          Cpl::Cli.show_info_header(config) if with_info_header
```

Comment:
Can we not just call `show_info_header` directly without `Cpl::Cli`, since we're inside of the class?

Reply:
Actually, in the current implementation (where I used `show_info_header` as a static method), we don't need `Cpl::Cli`.

Thanks for reminding.

The reason I didn't use a class instance variable was that I couldn't find a way to keep `show_info_header` private and still be able to call it without an explicit `Cpl::Cli` prefix. I still don't think this method should be a static method.

Reply:
You're right.

Maybe it would be even better to move `show_info_header` to a helper then, because it would be easier to call it. But the current way works too, so up to you. I'll approve the PR.

Reply:
Since we use it once, let's keep it here. Let's move it there when we need to. Agree?